### PR TITLE
Added recipe for r-rjags

### PR DESCRIPTION
**Pull request by micknudsen**

### CODE REVIEW

**Comment by micknudsen** [comment]
Travis CI build fails on Mac OS (same problem as in https://github.com/conda-forge/staged-recipes/issues/4320). Decided to skip Mac OS builds, but now CircleCI fails. I don't see any hints of what is wrong in the errors messages. Any ideas?

**Comment by bgruening** [comment]
@micknudsen I retriggered a build. Do you mind using these script for creating r-packages?
https://github.com/bgruening/conda_r_skeleton_helper

It will remove the comments from the recipe and also adds a few maintainers.

File: recipes/r-rjags/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  summary: Interface to the JAGS MCMC library.
+  license_family: GPL2
+
+extra:
+  recipe-maintainers:
+    - micknudsen
+
+# The original CRAN metadata for this package was:
+
+# Package: rjags
```
Comment (bgruening) [inline recipes/r-rjags/meta.yaml]: please remove the comments

File: recipes/r-rjags/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+
+$R CMD INSTALL --build .
+
+# Add more build steps here, if they are necessary.
```
Comment (bgruening) [inline recipes/r-rjags/build.sh]: please remove the comments

File: recipes/r-rjags/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+about:
+  home: http://mcmc-jags.sourceforge.net
+  license: GPL (== 2)
+  summary: Interface to the JAGS MCMC library.
+  license_family: GPL2
+
+extra:
+  recipe-maintainers:
+    - micknudsen
```
Comment (bgruening) [inline recipes/r-rjags/meta.yaml]: If you like, please include a few more maintainers: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

Comment (micknudsen) [reply recipes/r-rjags/meta.yaml]: Should I just add everybody on the list?

**Comment by micknudsen** [comment]
@bgruening Do you have any idea why CircleCI fails? I don't see any clues in the error message.

**Comment by bgruening** [comment]
It has picked up the wrong R version. I restarted, let's see. This bug was in theory fixed yesterday.

File: recipes/r-rjags/meta.yaml (hunk 3 above)
Comment (bgruening) [reply recipes/r-rjags/meta.yaml]: Yes, please.

**Comment by micknudsen** [comment]
@bgruening Hmmm. CircleCI is still failing. It is using R 3.4.2, which is the newest one available. Is that somehow wrong?

**Comment by bgruening** [comment]
Yes, this should not happen anymore. R 3.4.2 comes from the default channel. conda-forge does not have this version. I'm not sure why this happens here, it should be fixed since yesterday. Other PRs are also working now.

I will work on this https://github.com/conda-forge/r-coda-feedstock/pull/1 and then restart this one once again.

**Comment by micknudsen** [comment]
@bgruening Would you mind restarting the build whenever you have the time? Just wondering if the problem has somehow fixed itself by now. Thanks.

File: recipes/r-rjags/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+test:
+  commands:
+    - $R -e "library('rjags')"
+
+about:
+  home: http://mcmc-jags.sourceforge.net
+  license: GPL (== 2)
+  summary: Interface to the JAGS MCMC library.
+  license_family: GPL2
```
Comment (bgruening) [inline recipes/r-rjags/meta.yaml]: license_file is missing as used here: https://github.com/bgruening/conda_r_skeleton_helper/blob/master/extra.yaml

Comment (micknudsen) [reply recipes/r-rjags/meta.yaml]: I don't see any license file included in rjags_4-6.tar.gz. What should one include then?

Comment (micknudsen) [reply recipes/r-rjags/meta.yaml]: @bgruening Could you elaborate a bit on what is missing? I looked at random R package at conda-forge (https://github.com/conda-forge/r-survey-feedstock/blob/master/recipe/meta.yaml), and that doesn't have a license_file entry either.

Comment (micknudsen) [reply recipes/r-rjags/meta.yaml]: @bgruening Solved it. You linked to the wrong file which left me quite confused :-)

**Comment by micknudsen** [comment]
Does this recipe need more work? I am hoping that it can be merged soon, so I can start working on a bioconda recipe which depends on it.

**Pull request merged**